### ISSUE TITLE
readme should have more info about terminus

### ISSUE THREAD

**Issue by insyri** [JOBBIN9422/TerminusDotNET#44]
To a foreigner, assuming they have no idea what the original terminus is, would have no idea what terminusdotnet does without going through the source code/toying with the bot (invite link is non-existent; if it is on the repo, it should be easier to find). I suggest expanding the readme to the extent of what kind of things it does or at least give a hyperlink if there is explanatory text for the original terminus (reading the [original terminus](https://github.com/ct-bess/Terminus) I couldn't find it's purpose other than the setup). To be fair, the original terminus in it's [readme](https://github.com/ct-bess/Terminus#introduction) did have a sentence that explained it purpose, however, personally I think it is vague in the word 'react' which can mean many different things (can be ping/pong type, calculator, etc):
> This project shows the capability of a bot to react to messages sent within a group.

TL;DR: readme should explain what the bot does and not just what it is.

Labels: enhancement documentation

**Comment by ct-bess** [comment]
dude

**Comment by insyri** [comment]
i.. i don't know
